easy-connect.h

```diff
@@ -61,8 +73,15 @@ NetworkInterface* easy_connect(bool log_messages = false) {
         output.printf("[EasyConnect] Using WiFi (ESP8266) \r\n");
         output.printf("[EasyConnect] Connecting to WiFi..\r\n");
     }
-    connect_success = esp.connect(MBED_CONF_APP_ESP8266_SSID, MBED_CONF_APP_ESP8266_PASSWORD);
+    connect_success = esp.connect(MBED_CONF_APP_WIFI_SSID, MBED_CONF_APP_WIFI_PASSWORD);
```

Comment:
If we really want to tweak the codelines down, we could use one common wifi.connect and always give the 3 parameters. That works also with the esp8266.